src/genie/libs/parser/ios/asr901/tests/ShowEnvironment/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,72 @@
+expected_output = {
+    "Fan": {
```

Comment:
in schema, we usually don't use upper case for key name. 
```suggestion
    "fan": {
```